Tribler/Test/API/test_seeding.py

```diff
@@ -36,6 +36,7 @@ def setUpPreSession(self):
         super(TestSeeding, self).setUpPreSession()
 
         self.config.set_libtorrent(True)
+        self.config.set_megacache(True)
```

Comment:
@NielsZeilemaker I made `SearchManager` to be always initialized and it depends on ChannelDBHandler, so that's why I have to add enabled megacache in a lot of tests.